bundles/org.openhab.binding.panamaxfurman/README.md

```diff
@@ -0,0 +1,68 @@
+# PanamaxFurman Binding
+
+Binding for Panamax/Furman power conditioners which have the BlueBOLT-CV1 or BlueBOLT-CV2 interface card installed.
+
+## Supported Things
+
+- `telnet`: Accesses the telnet interface of the BlueBOLT-CV1 or BlueBOLT-CV2 interface card
+
+## Discovery
+
+Discovery is not supported by the Panamax/Furman power conditioner devices.
+
+## Binding Configuration
+
+No configuration required
+
+## Thing Configuration
+
+### `telnet` Thing Configuration
+
+| Name            | Type    | Description                           | Default | Required | Advanced |
+|-----------------|---------|---------------------------------------|---------|----------|----------|
+| address        | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+
+
+## Channels
+
+| Channel | Type   | Read/Write | Description                 |
+|---------|--------|------------|-----------------------------|
+| Brand | String | R         | Brand of the power conditioner|
+| Model | String | R         | Model of the power conditioner|
+| Firmware Version | String | R         | Firmware version of the power conditioner|
```

Comment:
Since Brand, Model and Firmware Version are just static metadata (rather than say a voltage measurement) that will not be useful in rules, these do not need to be a channel. They can be a Thing Property instead. Here is an example of how Think Properties look for Roku when displaying similar data:
![image](https://user-images.githubusercontent.com/26532542/172248735-3e67c641-c937-4ce6-9aad-be6cda28b3b6.png)